models/release.go

```diff
@@ -6,13 +6,15 @@ package models
 
 import (
 	"fmt"
+	"path"
 	"sort"
 	"strings"
 	"time"
 
 	"code.gitea.io/git"
 	"code.gitea.io/gitea/modules/process"
 	"code.gitea.io/gitea/modules/setting"
+	"code.gitea.io/gitea/modules/util"
 	api "code.gitea.io/sdk/gitea"
 	"github.com/go-xorm/xorm"
 )
```

Comment:
Structure is:
bultin imports
empty line
gitea imports
empty line
dependency imports